Source/Scene/AutoExposure.js

```diff
@@ -0,0 +1,432 @@
+
+        /**
+         * The minimum value used to clamp the luminance.
+         *
+         * @type {Number}
+         * @default 0.1
+         */
+        this.minimumLuminance = 0.1;
+
+        /**
+         * The maximum value used to clamp the luminance.
+         *
+         * @type {Number}
+         * @default 10.0
+         */
+        this.maximumLuminance = 10.0;
+    }
+
+    defineProperties(AutoExposure.prototype, {
+        /**
+         * Determines if this post-process stage is ready to be executed. A stage is only executed when both <code>ready</code>
+         * and {@link AutoExposure#enabled} are <code>true</code>. A stage will not be ready while it is waiting on textures
+         * to load.
+         *
+         * @memberof AutoExposure.prototype
+         * @type {Boolean}
+         * @readonly
+         */
+        ready : {
+            get : function() {
+                return this._ready;
+            }
+        },
+        /**
+         * The unique name of this post-process stage for reference by other stages.
+         *
+         * @memberof AutoExposure.prototype
+         * @type {String}
+         * @readonly
+         */
+        name : {
+            get : function() {
+                return this._name;
+            }
+        },
+
+        /**
+         * A reference to the texture written to when executing this post process stage.
+         *
+         * @memberof AutoExposure.prototype
+         * @type {Texture}
+         * @readonly
+         * @private
+         */
+        outputTexture : {
+            get : function() {
+                var framebuffers = this._framebuffers;
+                if (!defined(framebuffers)) {
+                    return undefined;
+                }
+                return framebuffers[framebuffers.length - 1].getColorTexture(0);
+            }
+        }
+    });
+
+    function destroyFramebuffers(autoexposure) {
+        var framebuffers = autoexposure._framebuffers;
+        if (!defined(framebuffers)) {
+            return;
+        }
+
+        var length = framebuffers.length;
+        for (var i = 0; i < length; ++i) {
+            framebuffers[i].destroy();
+        }
+        autoexposure._framebuffers = undefined;
+
+        autoexposure._previousLuminance.destroy();
+        autoexposure._previousLuminance = undefined;
+    }
+
+    function createFramebuffers(autoexposure, context) {
+        destroyFramebuffers(autoexposure);
+
+        var width = autoexposure._width;
+        var height = autoexposure._height;
+
+        var pixelFormat = PixelFormat.RGBA;
+        var pixelDatatype = context.halfFloatingPointTexture ? PixelDatatype.HALF_FLOAT : PixelDatatype.FLOAT;
+        var sampler = new Sampler({
+            wrapS : TextureWrap.CLAMP_TO_EDGE,
+            wrapT : TextureWrap.CLAMP_TO_EDGE,
+            minificationFilter : TextureMinificationFilter.NEAREST,
+            magnificationFilter : TextureMagnificationFilter.NEAREST
```

Comment:
Probably. Where would you sample? Between two pixels? That would also impact the samples count.

Since this isn't used right now, I'd put that in future work. I'll update the list above.